src/components/referenceChecking.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <p class='comment_user_text' v-if='isReferenceToUser'>
+    <span class='comment_user_text_first_word'>@{{ firstWord }}</span>
+    {{ commentText.slice(firstSpaceIndex + 1) }}
+  </p>
+  <p class='comment_user_text' v-else>{{ commentText }}</p>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+
+export default {
+  name: 'ReferenceChecking',
+  props: {
+    commentText: {
+      type: String,
+      required: true,
+    },
+  },
+  computed: {
+    ...mapState({
+      allAccountsList: (state) => state.allAccounts.allAccountsList,
+    }),
+    firstSpaceIndex() {
+      return this.commentText.indexOf(' ');
+    },
+    firstWord() {
+      if (this.commentText.startsWith('@')) {
+        return this.commentText.substring(1, this.firstSpaceIndex);
+      } else {
+        return false;
+      }
+    },
```

Comment:
there's still a problem with this function. it can return values of different types: string or boolean. it is inconvenient to work with such functions, all functions should have consistent return types. 

another problem is that I can write something like `@@@@@@@nickname` and this function will return it even though it doesn't look correct. I would recommend to use regular expression here to check for username pattern